universal-login-commons/lib/core/utils/authorisation.ts

```diff
@@ -0,0 +1,24 @@
+import {utils} from 'ethers';
+
+import {CancelAuthorisationRequest} from '../models/authorisation';
+
+export const hashCancelAuthorisationRequest =
+  (cancelAuthorisationRequest: CancelAuthorisationRequest): string => {
+    const payloadString = JSON.stringify(cancelAuthorisationRequest);
```

Comment:
Let's use `solidityKeccak256` for serialization, as:
- it is deterministic
- it is consistent with other serializations in the application

See [example usage](https://github.com/UniversalLogin/UniversalLoginSDK/blob/2942e5c06886fd19b260bd61215b59086b3d1098/universal-login-commons/lib/core/utils/calculateSignature.ts#L11).